index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio | Microverse Student Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <main class="wrapper">
+        <header class="header">
+            <div class="toolbar">
+                <a class="logo" href="#" aria-label="Homepage">>DS_</a>
+                <img src="./images/menu-btn.svg" alt="">
+            </div>
+
+            <div class="headline">
+                <h1 class="intro">I’m Selase 👋 Glad to see you!</h1>
+                <p class="description">
+                    I’m a software developer! I can help you build a product , feature or website. Look through some of
+                    my work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                    to contact me.
+                </p>
+                <p class="social-media">Let's connect</p>
+                <div class="socials">
+                    <a class="social-icon" href="#" aria-label="twitter">
+                        <img src="./images/twitter.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="Linkedin">
+                        <img src="./images/Linkedin.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="medium">
+                        <img src="./images/medium.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="Github">
+                        <img src="./images/github.svg" alt="">
+                    </a>
+                    <a class="social-icon" href="#" aria-label="AngelList">
+                        <img src="./images/AngelList.svg" alt="">
+                    </a>
```

Comment:
Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)